### DISCUSSION TITLE
Announcing: Project templates updates, Sticky group headers, and more!

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#66157] [Projects and Issues]
Today's Issues & Projects update brings brings you updates to project templates and historical charts, sticky group headers, and color improvements!

## :link: Project templates updates

We're continuing to make improvements to [project templates for organizations](https://docs.github.com/en/issues/planning-and-tracking-with-projects/managing-your-project/managing-project-templates-in-your-organization) (public beta). 👏

You can now view the project template that was used when a project was created. Once a template is used to create a new project, you can find and link back to the template from the project settings page in the "Templates" section. This allows you to reference the history of how a project was created and view any improvements made to the template since then. 

![templates settings page](https://user-images.githubusercontent.com/101840513/266406182-67db33f0-2581-47ed-bcc3-372a6e6f06ba.jpg)

Check out the [documentation](https://cli.github.com/manual/gh_project_mark-template) for more details, and as well as the project CLI [documentation](https://cli.github.com/manual/gh_project) to see all the possibilities of interacting with your projects from the terminal.

Be sure to drop a note in the feedback [discussion](https://github.com/orgs/community/discussions/54576) to let us know how we can continue to improve project templates.

## :scroll: Sticky group headers

Group headers are now sticky when scrolling through your project view. For example, if you have swimlanes on your board, scrolling to other columns and items will maintain the group name in view, making it easy for you to keep your place.

https://github.com/community/community/assets/92824518/735f7a3b-03aa-4071-aa4c-7a5decc51dc1

## 🎨 Updated colors for single select fields

The colors for single select fields have been updated, so you'll now see the same colors within the field picker and on your project views.

![single select field colors](https://user-images.githubusercontent.com/101840513/266406789-9c8b0a33-3e04-4c3e-81bb-cacca6aaa940.jpg)

## ➕ Create issues in board repository groups

You can now create issues when grouped by `Repository` on the board layout. Click `Create new issue` or start typing the title to get started.

![creating issue in repository groups](https://user-images.githubusercontent.com/101840513/266407043-add1e9db-86d0-4e1f-8811-490781ec46e9.jpg)

## :bar_chart: Updates to historical charts

[Historical charts](https://docs.github.com/en/issues/planning-and-tracking-with-projects/viewing-insights-from-your-project/about-insights-for-projects#about-historical-charts) (available for GitHub Team and Enterprise Cloud plans) now show the state changes of your project items over time, allowing you to see how items have been opened and closed over time. This allows you to visualize the progress of your items over time, showing how much work has been completed and how much is left to do.

![historical chart](https://user-images.githubusercontent.com/101840513/266407335-d12db089-6d87-4050-b211-48aeecd135b3.jpg)

## :sparkles: Bug fixes and improvements
* Included the `Slice by` field configuration when you copy a project or use a project template
* Improved the contrast of the roadmap today marker
* Fixed the alignment between the filter bar and board columns
* Fixed a bug where you could not copy a project or use a template if there were invalid disabled workflows
* Fixed a bug where the view configuration menu alignement shifted when resizing a window
* Updated the error message when setting a large column limit on board columns

See how to use GitHub for project planning with [GitHub Issues](http://github.com/features/issues), check out what's on the [roadmap](https://github.com/orgs/github/projects/4247/views/7), and learn more in the [docs](https://docs.github.com/issues).

**Comment by devtech5789** [comment]
It seems like you're making an announcement about updates to a project or software. Providing clear and informative announcements is essential for keeping your users or team members informed. If you have more details or specific information you'd like to share, please feel free to provide more context or ask any questions you may have about making announcements. I'm here to help with any information or assistance you need.

**Comment by MirLeigh** [comment]
Is the template feature available to all users? Or is it only the org admin that can create issue templates?

**Reply by queenofcorgis** [reply]
@MirLeigh - in the docs:

> People with admin permissions for a project in an organization can set the project as a template. People with admin or write permissions for a project in an organization can copy the project and set the copied project as a template.
